deepchecks/vision/checks/data_integrity/abstract_property_outliers.py

```diff
@@ -109,12 +109,9 @@ def update(self, context: Context, batch: BatchWrapper, dataset_kind: DatasetKin
                 labels = [[label_per_image] for label_per_image in batch.numpy_labels]
             else:
                 labels = batch.numpy_labels
-            if not isinstance(property_values[0], t.Sequence):  # TODO: change after changing properties api
-                property_values = [[x] for x in property_values]
 
-            self._properties_results[prop_name].extend(property_values)
             self._images_uuid += batch.numpy_image_identifiers
-            self._update_lowest_highest_property_values(batch.numpy_images, labels, property_values, prop_name)
+            self._cache_property_values_and_images(batch.numpy_images, labels, list(property_values), prop_name)
```

Comment:
`list(`
why you care? is sequence not enough

Reply:
its for user defined custom properties